en/plugins.rst

```diff
@@ -280,6 +280,54 @@ autoloader once you've created your plugin:
 
     $ php composer.phar dumpautoload
 
+.. _plugin-routes:
+
+Plugin Routes
+=============
+
+Plugins can provide routes files containing their routes. Each plugin can
+contain a **config/routes.php** file. This routes file can be loaded when the
+plugin is added, or in the application's routes file. To create the
+ContactManager plugin routes. Put the following into
+**plugins/ContactManager/config/routes.php**::
+
+    <?php
+    use Cake\Routing\Route\DashedRoute;
+    use Cake\Routing\Router;
+
+    Router::plugin(
+        'ContactManager',
+        ['path' => '/contact-manager'],
+        function ($routes) {
+            $routes->fallbacks(DashedRoute::class);
```

Comment:
May be add one plugin specific rule plus the fallbacks? Might help envision what one can do with it.